internal/controller/vrg_volrep.go

```diff
@@ -2730,3 +2719,39 @@ func PruneAnnotations(annotations map[string]string) map[string]string {
 
 	return result
 }
+
```

Comment:
There is no conflict checking function for volrep?

Reply:
Updated the code: https://github.com/RamenDR/ramen/pull/1855/files#diff-a3c633f01e052ce1a42ea903ff76debacd956585919969d1f164ff88a7028f44R2758